server/lib/nl/fulfillment/triple.py

```diff
@@ -109,7 +126,9 @@ def _add_multi_entity_overview(state: PopulateState) -> bool:
   for prop_list in properties(entity_dcids).values():
     prop_set.update(prop_list)
   prop_list = [f'{_OUT_ARROW}{prop}' for prop in sorted(list(prop_set))]
-  cv = ChartVars(props=prop_list)
+  chart_title = _OVERVIEW_TITLE.format(
+      entity=_get_entity_string(state.uttr.entities))
```

Comment:
Should this happen in config builder?  (typically fulfillers don't concern with formatting names)

Reply:
There's a TODO to move getting title to config builder. There will need to be at least one more PR for title change when we have the actual title templates, so was going to try to make the move when we do that!